templates/user/linux/default/installer.erb

```diff
@@ -0,0 +1,11 @@
+user_check() {
+  getent passwd "$1" > /dev/null 2>&1
+}
+
+user_create() {
+  useradd "$1"
```

Comment:
Did some digging. The `useradd` tool comes from http://pkg-shadow.alioth.debian.org/. The `--system` flag appears to have been added in Feb 2008:

> 2008-02-19  Nicolas François  <nicolas.francois@centraliens.net>
...
       * NEWS, src/newusers.c, src/useradd.c, src/groupadd.c: Added new
       option -r, --system for system accounts in useradd, groupadd, and
       newusers.
```

This may mean CentOS 5 users won't have this flag.